### PR TITLE
-keyalg RSA is needed or you'll get SSLHandshakeException: no cipher …

### PR DESCRIPTION
**Pull request by StephanWald**
This took me a short while to figure out - I just copy and pasted the keytool statement from the comment line and got SSLHandshakeException. If you add -keyalg RSA all works as described. I suggest adding this to the sample line so others who try this out will be on track immediately

### CODE REVIEW

File: src/main/example/SSLServerExample.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ public static void main( String[] args ) throws Exception {
 
 		// load up the key store
 		String STORETYPE = "JKS";
-		String KEYSTORE = "keystore.jks";
+		String KEYSTORE = "D:\\GitHub\\Java-WebSocket\\src\\main\\example\\keystore.jks";
```
Comment (marci4) [inline src/main/example/SSLServerExample.java]: Could you please revert this change?

**Comment by StephanWald** [comment]
sorry
